src/components/table/__snapshots__/table.spec.js.snap

```diff
@@ -0,0 +1,1931 @@
+                                      "primary": "#26A826",
+                                      "secondary": "#006300",
+                                      "success": "#00B000",
+                                      "tertiary": "#004400",
+                                      "warning": "#E96400",
+                                      "white": "#FFFFFF",
+                                      "whiteMix": "#E5F4E5",
+                                    },
+                                    "disabled": Object {
+                                      "background": "#E5EAEC",
+                                      "border": "#CCD6DA",
+                                      "button": "#E5EAEC",
+                                      "buttonText": "rgba(0,0,0,.2)",
+                                      "disabled": "rgba(0,0,0,0.55)",
+                                      "input": "#F2F4F5",
+                                      "text": "rgba(0,0,0,0.3)",
+                                    },
+                                    "icon": Object {
+                                      "focus": "#335B6D",
+                                    },
+                                    "name": "base",
+                                    "shadows": Object {
+                                      "cards": "0 3px 3px 0 rgba(0,20,29,0.2), 0 2px 4px 0 rgba(0,20,29,0.15)",
+                                      "depth1": "0 5px 5px 0 rgba(0,20,29,0.2), 0 10px 10px 0 rgba(0,20,29,0.1)",
+                                      "depth2": "0 10px 20px 0 rgba(0,20,29,0.2), 0 20px 40px 0 rgba(0,20,29,0.1)",
+                                      "depth3": "0 10px 30px 0 rgba(0,20,29,0.1), 0 30px 60px 0 rgba(0,20,29,0.1)",
+                                      "depth4": "0 10px 40px 0 rgba(0,20,29,0.04), 0 50px 80px 0 rgba(0,20,29,0.1)",
+                                    },
+                                    "table": Object {
+                                      "header": "#335B6D",
+                                      "hover": "#E5EAEC",
+                                      "primary": "#F2F4F5",
+                                      "secondary": "#CCD6DA",
+                                      "selected": "#D8E0E3",
+                                      "sizes": Object {
+                                        "compact": "24px",
+                                        "large": "48px",
+                                        "medium": "40px",
+                                        "small": "32px",
+                                      },
+                                      "tertiary": "#19475B",
+                                      "zebra": "#F9FAFB",
+                                    },
+                                    "text": Object {
+                                      "color": "rgba(0,0,0,0.9)",
+                                      "placeholder": "rgba(0,0,0,0.3)",
+
```

Comment:
please remove the theme object from this snapshot by using either Enzyme's `shallow` or `TestRenderer.create` method

Reply:
I've removed this test, `shallow` still includes the theme object and `TestRenderer` will not work with the `CssTransitionGroup`